fgpyo/util/distance.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
I would put these methods in `sequence.py`, as I think we'll want to slowly move away from having a `util` sub-module where things just get shoved.